<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Messenger</class>
 <widget class="QMainWindow" name="Messenger">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>822</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Messenger</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background: rgb(63, 63, 96);
color:black;</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background:rgb(63, 63, 96);</string>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(63, 63, 96);
color:black;</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="main">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <widget class="QSplitter" name="splitter">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>9</y>
          <width>771</width>
          <height>532</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QTabWidget" name="tabWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QTabWidget:{
 background:  #73739b;
}

QPushButton:enabled
{
                font: 13pt &quot;Open Sans Semibold&quot;;
                background:  #73739b;
                border-radius: 3px;
                min-height:25px;
                padding: 5px;
                outline: 0;
            color: white;
}

QPushButton:hover
{
                background: #7a7aa1;
}

QPushButton:pressed
 {
        background: #8f8fb3;
}</string>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="Find">
          <property name="styleSheet">
           <string notr="true">background:rgb(63, 63, 96);
</string>
          </property>
          <attribute name="title">
           <string>Find</string>
          </attribute>
          <widget class="QWidget" name="layoutWidget">
           <property name="geometry">
            <rect>
             <x>9</x>
             <y>10</y>
             <width>258</width>
             <height>489</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLineEdit" name="FindClient">
                <property name="minimumSize">
                 <size>
                  <width>181</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>181</width>
                  <height>22</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">color:white;</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="FindClientButton">
                <property name="minimumSize">
                 <size>
                  <width>61</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>61</width>
                  <height>21</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true"> background:  #73739b;
color:black;</string>
                </property>
                <property name="text">
                 <string>Find</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QListWidget" name="listClients">
              <property name="minimumSize">
               <size>
                <width>252</width>
                <height>0</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">color:rgb(255, 255, 255);</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QWidget" name="Friends">
          <property name="styleSheet">
           <string notr="true">background:rgb(63, 63, 96);
color:black;</string>
          </property>
          <attribute name="title">
           <string>Friends</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QListWidget" name="listFriends"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="Settings">
          <property name="styleSheet">
           <string notr="true">background:rgb(63, 63, 96);
color:black;</string>
          </property>
          <attribute name="title">
           <string>Settings</string>
          </attribute>
          <widget class="QCheckBox" name="SaveSizeWindow">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>111</width>
             <height>35</height>
            </rect>
           </property>
           <property name="text">
            <string>Save size window</string>
           </property>
          </widget>
         </widget>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="chat">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>491</width>
              <height>471</height>
             </size>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true"> background:rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLineEdit" name="input">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>371</width>
                <height>51</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>371</width>
                <height>51</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"> background:  rgb(255, 255, 255);</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="send">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>111</width>
                <height>51</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>111</width>
                <height>51</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"> background:  #73739b;
color:black;</string>
              </property>
              <property name="text">
               <string>Send</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="Login">
       <property name="styleSheet">
        <string notr="true">background:rgb(63, 63, 96);</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QWidget" name="widget_1" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>98</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Expanding</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="minimumSize">
                  <size>
                   <width>551</width>
                   <height>21</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>551</width>
                   <height>21</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QLabel
{
        font: 13pt &quot;Open Sans Light&quot;
        color:blue;
}</string>
                 </property>
                 <property name="text">
                  <string>Username:</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                 </property>
                 <property name="margin">
                  <number>0</number>
                 </property>
                 <property name="indent">
                  <number>0</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="login_username">
                 <property name="minimumSize">
                  <size>
                   <width>551</width>
                   <height>35</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>551</width>
                   <height>35</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QLineEdit
{
        font: 11pt &quot;Open Sans&quot;;
        border:1px solid  grey ;
        min-height: 33px;
        background: white;
       color:black;
}

QLineEdit:focus
{
        border:1px solid  #161626 ;
} </string>
                 </property>
                 <property name="text">
                  <string>User</string>
                 </property>
                 <property name="maxLength">
                  <number>20</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_6">
                 <property name="minimumSize">
                  <size>
                   <width>551</width>
                   <height>21</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>551</width>
                   <height>21</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QLabel
{
        font: 13pt &quot;Open Sans Light&quot;
color:blue
;
}</string>
                 </property>
                 <property name="text">
                  <string>Password:</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                 </property>
                 <property name="indent">
                  <number>0</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="login_password">
                 <property name="minimumSize">
                  <size>
                   <width>551</width>
                   <height>35</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>551</width>
                   <height>35</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QLineEdit
{
        font: 11pt &quot;Open Sans&quot;;
        border:1px solid  grey ;
        min-height: 33px;
        background: white;
        color:black;
}

QLineEdit:focus
{
        border:1px solid  #161626 ;
} </string>
                 </property>
                 <property name="inputMethodHints">
                  <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
                 </property>
                 <property name="text">
                  <string>1111</string>
                 </property>
                 <property name="maxLength">
                  <number>30</number>
                 </property>
                 <property name="echoMode">
                  <enum>QLineEdit::Password</enum>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <property name="bottomMargin">
                  <number>5</number>
                 </property>
                 <item>
                  <widget class="QPushButton" name="login_button">
                   <property name="styleSheet">
                    <string notr="true">QPushButton:enabled
{
                font: 13pt &quot;Open Sans Semibold&quot;;
                background:  #73739b;
                border-radius: 3px;
                min-height:25px;
                padding: 5px;
                outline: 0;
            color: white;
}

QPushButton:disable
{
                font: 13pt &quot;Open Sans Semibold&quot;;
                background:  gray;
                border-radius: 3px;
                min-height:25px;
                padding: 5px;
                outline: 0;
            color: white;
}


QPushButton:hover
{
                background: #7a7aa1;
}

QPushButton:pressed
 {
        background: #8f8fb3;
}</string>
                   </property>
                   <property name="text">
                    <string>Log in</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="login_signup">
                   <property name="styleSheet">
                    <string notr="true">QPushButton:enabled
{
                font: 13pt &quot;Open Sans Semibold&quot;;
                background:  #73739b;
                border-radius: 3px;
                min-height:25px;
                padding: 5px;
                outline: 0;
                color: white;
}

QPushButton:hover
{
                background: #7a7aa1;
}

QPushButton:pressed
 {
        background: #8f8fb3;
}</string>
                   </property>
                   <property name="text">
                    <string>Sign up</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QLabel" name="login_error">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>551</width>
                   <height>21</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>551</width>
                   <height>21</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">font: 10pt &quot;Open Sans Light&quot;;
color:black;</string>
                 </property>
                 <property name="text">
                  <string>Connection...</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>45</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>97</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
